doc/en/installation/common/post_install.rst

```diff
@@ -0,0 +1,82 @@
+To start the monitoring engine :
+
+1. On your web interface, go to **Configuration** ==> **Pollers**.
+2. Keep the default options and click on **Export configuration**.
+3. Select **Central** poller from the box input **Pollers**.
+4. Uncheck **Generate Configuration Files** and **Run monitoring engine debug (-v)**.
+5. Check **Move Export Files** and **Restart Monitoring Engine** with option **Restart** selected.
+6. Click on **Export** again.
+7. Log in to the ‘root’ user on your server.
+8. Start Centreon Broker ::
+
+     # systemctl cbd start
+
+9. Start Centreon Engine ::
+
+     # systemctl centengine start
+
+10. Start centcore :: 
+
+    # systemctl centcore start
+
+11. Start centreontrapd ::
+
+    # systemctl centreontrapd start
+
+Monitoring is now working. You can begin monitoring your IT system!
+
+To make services automatically start during system bootup run these commands
+on the central server: ::
+
+    # systemctl enable centcore
+    # systemctl enable centreontrapd
+    # systemctl enable cbd
+    # systemctl enable centengine
+
```

Comment:
ces commandes ont déjà été lancées plus haut dans la doc d'installation, a -t-on besoin de les retaper une fois que la supervision a démarré ?

Reply:
I suppose yes because I use include of file and I'm not sure that there will not impact